fanaSystem/fanaAuthenticator/utils.py

```diff
@@ -0,0 +1,26 @@
+from decouple import config
+from twilio.rest import Client
+from twilio.base.exceptions import TwilioRestException
+import random
+from fanaSystem.utils import generic_error_handler
+
+
+
+
+def generate_otp(length = 6):
+    try:
+        return ''.join([str(random.randint(0, 9)) for _ in range(length)])
+    except Exception as e:
+        raise e
+
+def send_otp(phone_number , otp ,length = 6):
+    try:
+        client = Client(config('SID') , config('AUTH_TOKEN'))
+        message = client.messages.create(
+            body=f'Your OTP is: {otp}',
+            from_=config('PH'),
+            to=phone_number
+        )
+        return {"success" : True , "message" : "Otp Sent Successfully !"}
+    except Exception as e:
+        raise e
```

Comment:
Ensure that the file ends with an empty line

Reply:
Done.